sample_formatted_code.do_not_merge.c

```diff
@@ -0,0 +1,562 @@
+// Formatted using
```

Comment:
I have only taken some selected things in here over to the code-style.rst.

Should probably go through this again and see if some of the settings need changing.

Reply:
I think I took over all the essential samples.